smurfs/src/components/App.css

```diff
@@ -1,28 +1,320 @@
+  pointer-events: none;
+  position: fixed;
+}
+
+@keyframes flying {
+  from {
+    top: 5%;
+  }
+  to {
+    top: 6%;
+  }
+}
+
+.smurfs {
+  position: absolute;
+  display: flex;
+  justify-content: space-between;
+  right: 0;
+  left: 0;
+  margin: 0 auto;
+  z-index: 400;
+  width: 1000px;
+  padding-right: 100px;
+  user-select: none;
+  pointer-events: none;
+  position: fixed;
+}
+
+.smurfette {
+  width: auto;
+  height: 515px;
+  z-index: 400;
+  user-select: none;
+  pointer-events: none;
+}
+
+.papa {
+  width: auto;
+  height: 500px;
+  left: 0%;
+  top: 22%;
+  z-index: 400;
+  user-select: none;
+  pointer-events: none;
+}
+
+.smurfCard {
+  text-align: center;
+  border-radius: 30px;
+  border: 3px dashed rgb(199, 238, 252);
   padding: 20px;
+  margin-bottom: 50px;
+}
+
+.smurfname {
+  font-size: 2rem;
+  margin: 0;
+  font-family: 'Luckiest Guy', cursive;
+  color: rgb(0, 140, 255);
+}
+
+.smurf-info {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto;
+  width: 300px;
+}
+
+.smurf-text {
+  font-size: 1.2rem;
+  font-weight: 700;
+  color: rgb(67, 165, 245);
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+input {
+  text-align: center;
+  width: 50%;
+  margin-bottom: 10px;
+  padding: 10px;
+  border: 2px solid rgb(121, 176, 248);
+  border-radius: 5px;
+}
+
+.submit-btn {
+  background-color: rgb(55, 228, 127);
+  box-shadow: 0px 10px 20px rgba(55, 228, 127, 0.568);
+  border: none;
+  width: 55%;
+  padding: 20px 10px;
+  border-radius: 8px;
+  color: white;
+  font-size: 1rem;
+  margin: 20px 5px;
+  cursor: pointer;
+  transition: 0.3s ease-in-out;
+}
+
+.del-button {
+  background-color: red;
+  border: none;
+  width: 120px;
+  padding: 8px 10px;
+  border-radius: 5px;
+  box-shadow: 0px 5px 20px rgba(255, 0, 0, 0.479);
   color: white;
+  margin: 0px 5px;
+  cursor: pointer;
+  transition: 0.3s ease-in-out;
+}
+
+.del-button:hover {
+  transition: 0.3s ease-in-out;
+  filter: brightness(90%);
 }
 
-.App-title {
-  font-size: 1.5em;
+.up-button {
+  background-color: rgb(0, 153, 255);
+  box-shadow: 0px 5px 20px rgba(0, 153, 255, 0.527);
+  border: none;
+  width: 120px;
+  padding: 8px 10px;
+  border-radius: 5px;
+  color: white;
+  margin: 0px 5px;
+  cursor: pointer;
+  transition: 0.3s ease-in-out;
 }
 
-.App-intro {
-  font-size: large;
+.up-button:hover {
+  transition: 0.3s ease-in-out;
+  filter: brightness(90%);
+}
+
+.add-button {
+  background-color: rgb(55, 228, 127);
+  box-shadow: 0px 10px 20px rgba(55, 228, 127, 0.568);
+  border: none;
+  width: 50%;
+  padding: 20px 10px;
+  border-radius: 8px;
+  color: white;
+  font-size: 1rem;
+  margin: 20px 5px;
+  cursor: pointer;
+  transition: 0.3s ease-in-out;
 }
 
-@keyframes App-logo-spin {
-  from { transform: rotate(0deg); }
-  to { transform: rotate(360deg); }
+.add-button:hover {
+  transition: 0.3s ease-in-out;
+  filter: brightness(90%);
```

Comment:
What impressive CSS and a large amount to get done in addition to all the CRUD logic too - that is super impressive. well done!